int32 dim
int32 ncomp
float64 dT
float64[] offset
float64[] priors
float64[] mus
float64[] sigmas
string source_fid
string target_fid